index.html

```diff
@@ -38,6 +40,195 @@ <h1>
+                            </ul>
+                            </div>
+                    <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+
+                    <div class="languages inline">
+                      <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                      </ul>
+                    </div>
+
+                    <a href="#" class="btn-one">See project</a>
+                  </li>
+                </ul>
+              </div>
+           </li>
+
+            <li class="cards">            
+              <div class="cards-container">
+                <ul>
+                  <li>
+                      <img src="images/card2.png" alt="Second Card">
+                  <li class="features">
+                    <h1><a class="head" href="">Multi-Post Stories</a></h1>
+                    <div class="information">
+                        <ul>
+                          <li class="option">Canopy</li>
+                          <li> <i class="fas fa-circle"></i> Back End Dev</li>
+                          <li><i class="fas fa-circle"></i> 2015</li>
+                        </ul>
+                    </div>
+                    <p> A daily selection of privtely personalized reads; no accounts or sign-ups required. </p>
+
+                    <div class="languages inline">
+                     <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                      </ul>
+                    </div>
+
+                    <a href="#" class="btn-one">See project</a>
+                  </li>
+                </ul>
+              </div>
+           </li>
+            <li class="cards">            
+              <div class="cards-container">
+                <ul>
+                  <li>
+                      <img src="images/card3.png" alt="Third Card">
+                  <li class="features">
+                    <h1><a class="head" href="">Tonic</a></h1>
+                    <div class="information">
+                      <ul>
+                        <li class="option">Canopy</li>
+                        <li> <i class="fas fa-circle"></i> Back End Dev</li>
+                        <li><i class="fas fa-circle"></i> 2015</li>
+                      </ul>
+                      </div>
+                    <p>A daily selection of privately personalized reads, no accounts or sign-ups required. </p>
+                    <div class="languages inline">
+                      <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                      </ul>
+                    </div>
+                    <a href="#" class="btn-one">See project</a>
```

Comment:
- All the buttons in the "work section"
 has a style on"enabled", "Hover", "Pressed", "Disabled"
 kindly check the design on Figma.
 Here is a screenshot of it:
 
![button-styles](https://user-images.githubusercontent.com/47100064/129455729-4745e6d2-552b-4e83-8680-8de7194e6a05.png)

Reply:
Hello Im afraid Im not clear with you are asking me to do please explain. you it took me alot of time to get this work done and review deadline is at midnight in my country what do i really need to change exactly

Reply:
I am referring to css pseudo-class.
Something like:
a:hover, a:active,....

Reply:
And by the way my Figma template is showing me the enabled buttons and thats exactly what i added in the project.